### ISSUE TITLE
Add option to set `<all_urls>` for `web_accessible_resources`'s `matches`

### ISSUE THREAD

**Issue by riywo** [extend-chrome/rollup-plugin-chrome-extension#282]
### Describe the problem

When creating an internal chrome extension, we sometime specify `*//*/*foo.json*` for `matches` of `content_scripts`. This works well when developing because RPCE only adds `<all_urls>` for `web_accessible_resources`'s `matches`.

However, when building production manifest, it only copies `matches` from `content_scripts`. This is an issue because Manifest V3 doesn't allow `*://*/*foo.json*` for `web_accessible_resources`'s `matches`:

> Only the origin is used to match URLs. Origins include subdomain matching. Paths are ignored.
https://developer.chrome.com/docs/extensions/mv3/manifest/web_accessible_resources/#manifest-declaration

### Describe the proposed solution

Since it's for internal chrome extension, it is ok to set `<all_urls>` in production manifest. It would be nice to have an option to set `<all_urls>`. Something like below:

```ts
chromeExtension({ manifest, webAccessbileResources: {
  matchAllUrls: true,
} })
```

### Alternatives considered

We can of course specify all the possible domains in `content_scripts`'s `matches` but we might miss something and need to keep updating.

### Importance

nice to have

**Comment by jacksteamdev** [comment]
@riywo Good idea, we should sanitize `web_accessible_resources` match patterns. :+1:

Would you like to make a PR?

**Comment by riywo** [comment]
Sure, I'll be able to send a PR in a week or so.

**Comment by riywo** [comment]
@jacksteamdev I'm trying to create an e2e test first, but `npx jest tests/e2e` on HEAD (i.e. without any modification) doesn't succeed on my macOS. Is there any caveat to run e2e tests?

**Comment by jacksteamdev** [comment]
@riywo Not that I know of. What error are you getting?

**Comment by riywo** [comment]
@jacksteamdev Hmm... I face multiple failures like this https://gist.github.com/riywo/c33d76b3e4abce574cfaaa94dcd2359d

**Comment by JhaAman** [comment]
Until this is fixed, you can copy/paste the `web_accessible_resources` from your `yarn dev` build, and when you run `yarn build` you manually replace `web_accessible_resources` with whatever you copied from the dev build. 

Or, I imagine OP's script automates that process.

**Comment by jacksteamdev** [comment]
@riywo @JhaAman This hit NPM in [@crxjs/vite-plugin@1.0.7](https://www.npmjs.com/package/@crxjs/vite-plugin) :rocket: 

Please let me know if it works for your use case! :pray:

**Comment by yudai-nkt** [comment]
I ran into the same problem and subscribed to this issues just a few hours before you closed this via PR. It seems working well for me at least!
